bloom-housing/bloom
# feat: geocoding geographic radius set up

# Pull Request Template

## Issue Overview

This PR addresses #3673


## Description

Adds fields for address validation. It will throw errors when no collectAddress provided, and then rest of fields will be validated when colllectAddress will be true, and radius just when `radius` option selected

## How Can This Be Tested/Reviewed?

On partners site go to `/settings`. Then add / edit item, then add/edit option. Under Additional Fields there will be new radio for `collect address`
- we shouldn't be able to submit with empty field
- check if fields updates correctly on backend.
- check if edit options prepopulates fields correctly

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
